ouroboros-consensus-cardano/test/tools-test/disk/config/genesis-conway.json

```diff
@@ -1,195 +1,26 @@
 {
-  "genDelegs": {},
```

Comment:
I think this file was just a copy paste of a previous era's genesis file, maybe plus the empty `genDelegs`?

Now it's contents is just what'll be in the conway genesis file.